packages/sdk/src/types/ContractAddress.ts

```diff
@@ -149,7 +176,8 @@ export function fromSerializable(value: Serializable): ContractAddress {
 }
 
 /**
- * Converts {@linkcode ContractAddress} into {@linkcode Serializable}
+ * Converts {@linkcode ContractAddress} into {@linkcode Serializable}.
+ * @deprecated Use the {@linkcode ContractAddress.toJSON} method instead.
```

Comment:
Why are we deprecating this? We could have the `toJSON` method call this, and provide both functions with no real downside.
To me, this seems like breaking change without any real value gained.

Reply:
This represents the integers as strings, whereas `toJSON` does bigints, so they're not the same. They were mostly there as a helper function for `toTypedJSON`.

Of course this depends on the resolution of the discussion in https://github.com/Concordium/concordium-node-sdk-js/issues/308#issuecomment-1976696107